packages/ui/src/helpers/authenticator/constants.ts

```diff
@@ -0,0 +1,130 @@
+/**
+ * This file contains helpers related to forms and input attributes.
+ */
+
+import { DefaultFormFieldOptions } from '../../types';
+import { countryDialCodes } from '../../i18n';
+
+export const defaultFormFieldOptions: DefaultFormFieldOptions = {
+  birthdate: {
+    label: 'Birthdate',
+    labelHidden: true,
+    placeholder: 'Birthdate',
+    type: 'date',
+    autocomplete: 'bday',
+    isRequired: true,
+  },
```

Comment:
Here are the default formFields value for each input.

Reply:
👍